recursion-schemes.cabal

```diff
@@ -83,7 +83,7 @@ library
 
   if flag(template-haskell)
     build-depends:
-      template-haskell >= 2.5.0.0 && < 2.21,
+      template-haskell >= 2.5.0.0 && < 2.22,
       th-abstraction   >= 0.4     && < 0.7
```

Comment:
It might be worth leaving a TODO here to bump the lower version bounds for `th-abstraction` to `>= 0.6` once the ecosystem has caught up, at which point we can remove the CPP in `Data.Functor.Foldable.TH`.

Reply:
Perhaps tracking this in a ticket would be better than an inline comment?

Reply:
Sure, wherever would be easiest to remember. Ultimately, I'm not really the maintainer here, so I'll let @gelisam make the call.

Reply:
I'm ok with either leaving it as-is without a comment nor a ticket, adding an inline comment, or adding a ticket.

Reply:
@RyanGlScott could you open a ticket for the bumping of bounds of `th-abstraction`?

Reply:
See #152.